### DISCUSSION TITLE
Relocating the Information Pane on the Artisan Roast Profile Printed Page?

### DISCUSSION THREAD

**Discussion by MONTSI94** [artisan-roaster-scope/artisan#1640] [General]
Hi,  I am looking for some guidance and advice on moving/relocating the Roast Information Pane that is on the completed Roast Profile Page.  [I am running Artisan Legacy 2.84 on a Mac which has macOS Catalina 10.15.7.]

The roast profile information iI am referring too is located in the upper right corner of the completed roast profile.  At times this information pane covers up the roast details on the graph.  I would like to be able to move/relocate this Information Pane to other locations on the roast profile page in order to have all roast graph details visible for review.  Is it possible to manipulate this Roast Information Pane of the Profile?   If so, how would I go about moving/relocating this Artisan Roast Information Pane?  (I am sure there is a more specific Artisan term for what I am referring to as the Roast Profile Information Pane, but I am not familiar with the proper nomenclature.)

I have attempted to paste a jpeg of the Roast Profile with the Information Pane circled in Red for reference....

![artisan Printed page w Info Window](https://github.com/user-attachments/assets/f9b596a9-6d01-450c-93c1-8e1d8a1a2573)

Thanks in advance for any help answering my question.
R/
Mike

**Comment by roasterdave** [comment]
Why not upgrade to Artisan v3.0.0?  Your OS is supported by the Artisan v3.0.0 macOS-legacy build.  

This feature is known as the Summary Statistics.  There are several enhancements to Summary Statistics in v3.0.0.  You can read all about it in the [v3.0.0 blog post ](https://artisan-roasterscope.blogspot.com/2024/08/artisan-v300.html)

To answer your specific question, the Summary Statistics cannot be relocated. By ticking `Auto` under Time Axis on Config>> Axes the graph will be automatically resized so important information is not hidden by the Summary Statistics.  To hide the Summary Statistics completely un-tick `Show Summary` on Config>> Statistics>> Stats Summary tab.  If you prefer to print the Summary Statistics separate from the profile graph use File>> Save Statistics.
